src/context.ts

```diff
@@ -19,6 +20,11 @@ import {
     type UserFromGetMe,
 } from "./platform.deno.ts";
 
+// === Util types
+export type MaybeArray<T> = T | T[];
+// deno-lint-ignore ban-types
+export type StringWithSuggestions<S extends string> = (string & {}) | S; // permits `string` but gives hints
```

Comment:
are you sure we want to export this?

Reply:
This isn|t exported from grammY, it is only exported because we want to provide the same suggestions in composer.ts.